library/ecp.c

```diff
@@ -167,14 +167,16 @@ struct mbedtls_ecp_restart_mul
 /*
  * Init restart_mul sub-context
  */
-static void ecp_restart_rsm_init( mbedtls_ecp_restart_mul_ctx *ctx )
-{
-    mbedtls_ecp_point_init( &ctx->R );
-    ctx->i = 0;
-    ctx->T = NULL;
-    ctx->T_size = 0;
-    ctx->state = ecp_rsm_init;
-}
+#define ecp_restart_rsm_init(ctx)               \
```

Comment:
I don't see this being used anywhere now - what am I missing?

Reply:
@hanno-arm has pointed out that what I am missing is https://github.com/Mbed-TLS/mbedtls/blob/development/library/ecp.c#L306 where the calls are generated by a macro